tests/test_elastic.py

```diff
@@ -21,3 +23,35 @@ def test_elasticsarch(elasticsearch):
 def test_random_port(elasticsearch_random):
     """Test if elasticsearch fixture can be started on random port."""
     assert elasticsearch_random.info()['status'] == 200
+
+
+@patch('pytest_elasticsearch.plugin.pytest.config')
+def test_ini_option_configuration(config):
+    """Test if ini and option configuration works."""
+    config.getoption.return_value = None
+    config.getini.return_value = '/tmp'
+
+    assert '/tmp' == plugin.elasticsearch_config()
+
+    config.getoption.return_value = '/tmp'
+
+    assert '/tmp' == plugin.elasticsearch_config()
```

Comment:
hwo can you be sure it's not the same value we're  returning here? (ie not from getini?) because that would also make this assert pass.